account_mass_reconcile/models/base_reconciliation.py

```diff
@@ -172,7 +172,7 @@ def create_write_off(self, lines, amount, amount_curr, same_curr):
 
         move = self.env["account.move"].create(
             {
-                "date": lines.env.context.get("date_p"),
+                "date": lines.env.context.get("date_p") or fields.Date.today(),
```

Comment:
```python
"date": lines.env.context.get("date_p", fields.Date.context_today(self)),
```

Please add a unit test for the use case when the date is not given.

Reply:
Done